qa/cephfs/conf/client.yaml

```diff
@@ -7,3 +7,4 @@ overrides:
         debug client: 20
         rados mon op timeout: 900
         rados osd op timeout: 900
+        client collect and send global metrics: false
```

Comment:
You don't want to disable metrics for all QA tests. This override should only go in `qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/v16.2.4.yaml`.